package.json

```diff
@@ -108,6 +109,7 @@
     "prism-sentry": "^1.0.2",
     "process": "^0.11.10",
     "prop-types": "^15.6.0",
+    "puppeteer": "^10.1.0",
```

Comment:
Why is this needed?

Reply:
Oh shoot this is from when I was trying to set up Storybook and got a compilation error. Just removed it :)